src/installers/helm.ts

```diff
@@ -162,7 +162,7 @@ error: E_COMMAND_FAILED`)
       tlsFlag = `-f ${destDir}values/tls.yaml`
     }
 
-    let command = `helm upgrade --install che --force --namespace ${flags.chenamespace} --set global.ingressDomain=${flags.domain} --set cheImage=${flags.cheimage} --set global.cheWorkspacesNamespace=${flags.chenamespace} ${multiUserFlag} ${tlsFlag} ${destDir}`
+    let command = `helm upgrade --install che --force --namespace ${flags.chenamespace} --set global.ingressDomain=${flags.domain} --set cheImage=${flags.cheimage} --set global.cheWorkspacesNamespace=${flags.chenamespace} ${multiUserFlag} ${tlsFlag} \"${destDir}`
```

Comment:
```suggestion
    let command = `"helm" "upgrade" "--install" "che" "--force" "--namespace" "${flags.chenamespace}" "--set" "global.ingressDomain=${flags.domain}" "--set cheImage=${flags.cheimage}" "--set global.cheWorkspacesNamespace=${flags.chenamespace}" "${multiUserFlag}" "${tlsFlag}" "${destDir}"`
```

Reply:
not working, replaced by using execa instead of execa.shell